packages/@lwc/template-compiler/src/parser/index.ts

```diff
@@ -956,7 +956,7 @@ function getTemplateAttribute(
 
     // Convert attribute name to lowercase because the location map keys follow the algorithm defined in the spec
     // https://wicg.github.io/controls-list/html-output/multipage/syntax.html#attribute-name-state
-    const location = element.location.attrs[name.toLowerCase()];
+    const location = element.location.attrs![name.toLowerCase()];
```

Comment:
It seems to me that `attrs` will always be defined, since we're in `getTemplateAttribute`:

https://github.com/salesforce/lwc/blob/95e685e82ae749e1af266fcd970025db2f501183/packages/%40lwc/template-compiler/src/parser/index.ts#L941-L945

Reply:
I looked into this locally and I had a squiggly red line there (assuming you were seeing the same). I was confused because its type indicated that the `attrs` property is always defined. The red line went away after I restarted the typescript server, so perhaps this change is unnecessary?

Reply:
That's odd, I got test failures on the command line before I added that `!`. Just double-checked, and it fails both `yarn build` and `yarn test` if I remove the `!`. My TS version is 4.4.4.

Reply:
My bad, I see that it became optional now:
```
export interface StartTagLocation extends Location {
    /**
     * Start tag attributes' location info
     */
    attrs?: AttributesLocation;
}
```
Something is up with my vim plugin, `:w` no longer saves the file so I wasn't testing what I thought I was testing after removing the `!`.